SmartDeviceLink/public/SDLVideoStreamingCapability.h

```diff
@@ -84,6 +100,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nullable, strong, nonatomic) NSNumber<SDLFloat> *scale;
 
+/**
+ The preferred frame rate per second of the head unit. The mobile application / app library may take other factors into account that constrain the frame rate lower than this value, but it should not perform streaming at a higher frame rate than this value.
+ @since SDL 7.1
+ */
```

Comment:
```suggestion
/**
 * The preferred frame rate per second of the head unit. The mobile application / app library may take other factors into account that constrain the frame rate lower than this value, but it should not perform streaming at a higher frame rate than this value.
 * {"num_min_value": 0, "num_max_value": 2147483647}
 *
 * @added in SmartDeviceLink 7.1.0
 */
```

Reply:
I just updated the docs with the generated documentation for the parameter

Reply:
Updated SDLVideoStreamingCapability.h based on the generated code. Thanks for the review.